app/src/main/java/chat/rocket/android/service/ServerInfoImpl.java

```diff
@@ -0,0 +1,98 @@
+package chat.rocket.android.service;
+
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import chat.rocket.android.realm_helper.RealmHelper;
+import chat.rocket.android.realm_helper.RealmStore;
+
+/**
+ * Backend implementation to store ServerInfo.
+ */
+public class ServerInfoImpl extends RealmObject {
```

Comment:
This is not an implementation of `ServerInfo`... but a `Realm` version of it. Maybe `RealmServerInfo`?